src/modules/cpu/kernel/crop_mirror_normalize.hpp

```diff
@@ -23,15 +23,12 @@ RppStatus crop_mirror_normalize_u8_u8_host_tensor(Rpp8u *srcPtr,
         RpptROIPtr roiPtrInput = &roiTensorPtrSrc[batchCount];
         compute_roi_validation_host(roiPtrInput, &roi, &roiDefault, roiType);
 
-        std::vector<float> mean(srcDescPtr->c), invStdDev(srcDescPtr->c);
         Rpp32u incrementPerImage = srcDescPtr->c * batchCount;
```

Comment:
Suggest change name to a more generic cmnParamLoc. Initialize to srcDescPtr->c * batchCount. Increment cmnParamLoc++ at end of the loop.